sbnci/Modules/PDS/pdsvalidation.fcl

```diff
@@ -0,0 +1,12 @@
+BEGIN_PROLOG
+
+sbnd_pdsvalidation_ci: 
+{
+   module_type:			"PDSValidation"
+   GenieGenModuleLabel:		"generator"
+   LArGeantModuleLabel:		"largeant"
+   OpFlashTPC1Label:            "opflashtpc1"
```

Comment:
This might be my limited knowledge of the PDS but do we not care about `opflashtpc0`?

Reply:
there may be a reason for this, but it escapes me at present. the experts can always add it ;)